doc/tutorials/sagemaker/sme_deploy_model.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>Can we reduce the instance type more, maybe to a ml.t3.medium?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I am yet to learn what it means, so will go with your suggestion.</p>